### PR TITLE
Set and show when a member packet was sent to a user

### PR DESCRIPTION
**Pull request by weedySeaDragon**
### PT Story:  Admin: when was a member packet sent?
https://www.pivotaltracker.com/story/show/163505574

Admin can view if/when a member packet was sent for a user in both the list of all users and on the individual user account pages.

### Changes proposed in this pull request:

1.  add column `date_membership_packet_sent` to `users` table
1. added sortable column to all users view for admin.  This column is a checkbox to for user that shows whether or not a packet has been sent.  When admin checks the box, the date is set to _today_ (or cleared if the box is unset)
2. add info to User view for admin. This shows text: that a packet has been sent on a particular date (= the date the admin checked the checkbox), or that it hasn't been sent.
3. add sortable column to all users view for admin
4. refactored the `show-user-details` feature:  pulled out the scenarios that only applied to an admin and put them into `features/admins/show-user-account-to-admin.feature`.  
   - I created the `/admins` directory so that we can move all features that only apply to _admins_ into there

5. **Two scenarios fail only intermittently on my machine:**
   - `create_account.feature`  Scenario: Creating an account
   - `edit_account.feature` Scenario: Editing an account
   The intermittent failure is at the same step in both scenarios: `And I should see t("show_in_english") image`  (= I should see the change language image).  I believe this is failing because the image may not always be visible; the (headless) browser window my be 'scrolled down' so that it is not in view.

   I commented out the step in both scenarios because I feel they were unnecessary in those scenarios.  We're already testing and verifying that the switch language image is there and working in other features.

#### All users: member packet column is sortable -- sorted 1

all-users-member-packet-sorted
---

#### All users: member packet column is sortable -- sorted 2

all-users-member-packet-sorted-2

---
#### User Account page: member packet has been sent

user-acct-pkt-sent

---

#### User Account page: member packet has not been sent

user-acct-pkt-not-sent

### Ready for review:
@patmbolger @thesuss @rmzse

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -162,6 +162,7 @@ sv:
         sign_in_count: &sign_in_count Inloggningar
         reset_password_sent_at: &user_reset_password_sent_at Information angående återställning av lösenord sändes
         member: &member Medlem
+        date_member_packet_sent: &date_member_packet_sent Datumpaket Skickat
```
Comment (thesuss) [inline config/locales/sv.yml]: Paket skickat:

**Comment by thesuss** [comment]
I dont see the code that actually puts the text on the view? 
So I can't comment in the right way, but the text, being an admin note, I think it should be %pre , not %p or  without tag. 

Just looked at what failed on cucumber and it seems I am not alone in not finding the text on that view :D

**Comment by patmbolger** [comment]
@weedySeaDragon The view changes are _not_ in this PR.  Am I missing something?

**Comment by weedySeaDragon** [comment]
After I pushed the recent changes (merge with current develop branch; actually SHOW the packet info..), the RSpec tests all passed. (build #2 https://semaphoreci.com/shf-project/shf-project/branches/pull-request-651/builds/2)
But the Cucumber build _stalled_! (The last task to run (most of) the cucumber tests _never even started after nearly 14 minutes!)

So I stopped the build and restarted it

Now the RSpec tests failed:  and the failures are all _db seeding tests!_   <sigh>  So. Weird.
And the 1 failing cucumber test has nothing to do with my PR.

**Comment by weedySeaDragon** [comment]
@patmbolger  and @thesuss --
After fixing the locale entry, the RSpec tests _pass_ and there is 1 failing Cucumber scenario:

> Failing Scenarios:
> cucumber features/create_membership_application.feature:382 # Scenario: Apply for membership: company number wrong length (no uploads) [SAD PATH]

```
Scenario: Apply for membership: company number wrong length (no uploads) [SAD PATH] # features/create_membership_application.feature:382
    Given I am on the "new application" page                                          # features/step_definitions/navigation_steps.rb:1
      # Create new company in modal
    And I click on t("companies.new.title")                                           # features/step_definitions/basic_steps.rb:7
    And I fill in the translated form with data:                                      # features/step_definitions/basic_steps.rb:65
      | companies.show.company_number | companies.show.email |
      | 00                            | kicki@immi.nu        |
      Ambiguous match, found 2 elements matching visible field "Company number" that is not disabled (Capybara::Ambiguous)
      ./features/step_definitions/basic_steps.rb:69:in `block (2 levels) in <top (required)>'
      ./features/step_definitions/basic_steps.rb:67:in `each'
      ./features/step_definitions/basic_steps.rb:67:in `/^I fill in the( translated)? form with data:$/'
      features/create_membership_application.feature:387:in `And I fill in the translated form with data:'
    And I click on t("companies.create.create_submit")                                # features/step_definitions/basic_steps.rb:7
    Then I should not see t("shf_applications.uploads.please_upload_again")           # features/step_definitions/assertion_steps.rb:81
```

Seems like it's one of those randomly failing cucumber/capybara tests.   The same steps work just fine in other scenarios in that feature.  (Creating a company with the new company modal; filling out the `company number` in the modal.)

If everything else looks good, can we squash and merge this despite this failure?  (Because the next story I need to work on (add the member packet info to the data that is exported into a CSV) depends on this _(cursed)_ one.

**Review by patmbolger — APPROVED** [review]
The test failure pattern is well-known and not caused by this PR, so good to go.

**Review by thesuss — APPROVED** [review]

**Pull request merged**